### PR TITLE
docs: add redirect schema step for android browser sign in

### PR DESCRIPTION
**Pull request by shuowu-okta**
Add `redirect schema` setup step for browser in sign android development.

### CODE REVIEW

File: browser-sign-in/README.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ export default {
 };
 ```
 
+For **Android** developement, please make sure `redirect schema` is properly added by following [Add redirect schema](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react-native#add-redirect-scheme) section from [Okta React Native](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react-native#okta-react-native) README.
```
Comment (swiftone) [inline browser-sign-in/README.md]: developement -> development

**Comment by shuowu-okta** [comment]
> I'm not sure I prefer this approach. This updates the readme to instruct the user to update the `appAuthRedirectScheme`, but there's no reason for the committed version of that file to have the incorrect value starting out. I've opened #32 as an alternative solution that updates the committed value in `build.gradle`, and provides more specific instructions in the readme in case the user chose a different Redirect URI

The value in `build.gradle` should be changed anyway to follow configurations from the created native application in your okta. For the doc, we decided to follow the SDK doc as the source of truth.

**Comment by CodingItWrong** [comment]
@shuowu-okta That doesn't match what the readme is currently saying. It doesn't say to pick a unique redirect URI for your application; it says to use `com.sampleapplication:/`. In both the "Okta Application" section and the `samples.config.js` section, it shows a hard-coded `com.sampleapplication:/`.

If the intent is for the user to choose their own unique redirect URI, those sections of the readme aren't currently communicating that, and they should be updated to instruct the user to pick their own redirect URI.

**Comment by blairnangle** [comment]
It's still not crystal clear to me as to what the value of `appAuthRedirectScheme` should be. I have seen several conflicting statements—[this forum answer](https://devforum.okta.com/t/issue-with-okta-android-sdk/6375/3), for example.

Could someone please clarify if it should be a reversed package name (`com.sampleapplication`), the aforementioned with a `:/` suffix, reversed Okta domain name (`com.okta.dev-943261`) or does it depend entirely on what is configured in the Okta organization?

More than happy to raise a pull request to update the docs once I understand it myself!

**Comment by shuowu** [comment]
@blairnangle in your case, it would be `com.okta.dev-943261`

**Comment by blairnangle** [comment]
@shuowu thanks for the prompt reply!

Will this always be the case, regardless of what I have configured for my application in my Okta org?

**Comment by shuowu** [comment]
@blairnangle it should be the reversed domain name (without `:/` suffix)

**Comment by blairnangle** [comment]
@shuowu thank you!

**Pull request merged**